### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by HChughtai**
Created basic instructions for people to perform their first pull request to the repo. This involves them adding their name to the Contributors.md file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+## How to do submit your first contribution
+
+1. Fork this repository to create a copy of this repository in your account
+2. Clone the forked repository to your local machine
+3. Create a branch with a name that describes your intended change
+4. Make the necessary changes and commit those changes
+   * Add your name to the `Contributors.md` file
+5. Push changes to your fork on GitHub
+6. Go to your repository on GitHub and click the `Compare & Pull Request` button
+7. Request that your change be merged.
```
Comment (jcmoggridge) [inline CONTRIBUTING.md]: Perhaps add click on 'Create Pull Request" button to step 7

**Review by jcmoggridge — APPROVED** [review]
Please see comment regarding step 7

File: CONTRIBUTING.md (hunk 1 above)
Comment (HChughtai) [reply CONTRIBUTING.md]: Good point, that would clear things up

**Comment by HChughtai** [comment]
Will do your change @jcmoggridge and then merge. Thanks!

**Pull request merged**